pinot-transport/src/main/java/com/linkedin/pinot/routing/RoutingTableSelector.java

```diff
@@ -26,7 +28,7 @@
    * @param realtimeTableName name of the realtime table (e.g. tableName_REALTIME)
    * @return
    */
-  public boolean shouldUseLLCRouting(final String realtimeTableName);
+  boolean shouldUseLLCRouting(final String realtimeTableName);
```

Comment:
scratch that, realized we have routingtableselector and another routing table implementation. one for LLC and one for HLC?